lib/validators/specValidator.js

```diff
@@ -92,6 +92,7 @@ class SpecValidator {
         options.definition = self.specInJson;
         options.jsonRefs = {};
         options.jsonRefs.relativeBase = self.specDir;
+        options.isPathCaseSensitive = self.options.isPathCaseSensitive;
```

Comment:
`self.options.isPathCaseSensitive;` Please document this option in the constructor documentation.

Reply:
Done